config/routes.rb

```diff
@@ -9,15 +9,27 @@
   #   get 'products/:id' => 'catalog#view'
   root 'task#index'
 
-  get '/' => 'task#index'
-
   get '/show/:id' => 'task#show'
 
   get '/tasks/new' => 'task#new'
-
   post '/tasks' => 'task#create'
 
-  delete 'tasks/:id' => 'task#destroy'
+  get '/tasks/:id' => 'task#edit'
+  patch '/tasks/:id' => 'task#update'
+
+  get '/tasks/comp_status/:id' => 'task#completed'
+  post '/tasks' => 'task#completed'
+
+  get '/tasks/ip_status/:id' => 'task#in_process'
+  post '/tasks' => 'task#in_process'
+
+  delete '/tasks/:id' => 'task#destroy'
+
+  get '/people' => 'person#index'
+
+  get '/people/show/:id' => 'person#show'
+
+  get '/people/:id/tasks' => 'person#persons_tasks'
 
```

Comment:
Most of your routes look good - I think there is probably some more cleanup you can do based on your further understanding of RESTful rotues now